src/signals/incident-management/containers/FilterTagList/index.js

```diff
@@ -15,33 +15,36 @@ const FilterWrapper = styled.div`
 const StyledTag = styled(Tag)`
   display: inline-block;
   margin: 0 5px 5px 0;
-`;
+  :first-letter {
+    text-transform: capitalize;
+  }`;
 
 export const allLabelAppend = ': Alles';
 
-const renderTag = (key, mainCategories, list) => {
-  let found = false;
-
-  if (list) {
-    found = list.find(i => i.key === key || i.slug === key);
-  }
-
-  let display = (found && found.value) || key;
-
-  if (!display) {
-    return;
-  }
+export const FilterTagListComponent = props => {
+  const {
+    tags,
+    dataLists,
+    categories: { main, sub },
+  } = props;
 
-  if (moment(display, 'YYYY-MM-DD', true).isValid()) {
-    display = moment(display).format('DD-MM-YYYY');
-  }
+  const map = {
+    ...dataLists,
+    maincategory_slug: main,
+    category_slug: sub,
+  };
 
-  const foundMain = mainCategories.find(i => i.key === key);
+  const mapKeys = useCallback(key => {
+    switch (key) {
+      case 'source':
+        return 'bron';
 
-  display += foundMain ? allLabelAppend : '';
+      default:
+        return key;
+    }
+  }, []);
 
-  // eslint-disable-next-line consistent-return
-  return (
+  const renderItem = useCallback((display, key) => (
```

Comment:
`useCallback` Without dependencies doesn't do anything. Looks like this function can be taken out of the context of the `FilterTagListComponent`